### PR TITLE
Rdm 7072-F-056-Submit event creation as Citizen

### PR DESCRIPTION
**Pull request by nitinsahu1**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/AM-440

### Change description ###
Bit more refactoring and tidy up.
Have created a new branch/PR as suggested by Nitish and added all review comments provided for PR #741 

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/aat/resources/features/F-056/S-582.td.json [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+          "classification": "PUBLIC",
+          "value": []
+        }
+      },
+      "after_submit_callback_response": null,
+      "callback_response_status_code": null,
+      "callback_response_status": null,
+      "delete_draft_response_status_code": null,
+      "delete_draft_response_status": null,
+      "security_classifications": {
```
Comment (shrivsksavit) [inline src/aat/resources/features/F-056/S-582.td.json]: please use the common inheritance feature if possible

File: src/aat/resources/features/F-056/S-582.td.json [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+          "OPTION3"
+        ],
+        "YesOrNoField": "Yes",
+        "EmailField": "testemail@gmail.com",
+        "TextField": "sometextfield",
+        "DateField": "2222-01-02",
+        "TextAreaField": "textarea",
+        "CollectionField": []
+      },
+      "data_classification": {
```
Comment (shrivsksavit) [inline src/aat/resources/features/F-056/S-582.td.json]: please use the common inheritance feature if possible

File: src/aat/resources/features/F-056/S-582.td.json [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+    "body" : {
+      "id": "[[ANY]]",
+      "jurisdiction": "BEFTA_JURISDICTION_2",
+      "state": "TODO",
+      "version": "[[ANY]]",
+      "case_type_id": "BEFTA_CASETYPE_2_1",
+      "created_date": "[[ANY]]",
+      "last_modified": "[[ANY]]",
+      "security_classification": "PUBLIC",
+      "case_data": {
```
Comment (shrivsksavit) [inline src/aat/resources/features/F-056/S-582.td.json]: please use the common inheritance feature if possible

File: src/aat/resources/features/F-056/S-582.td.json (hunk 1 above)
Comment (nitinsahu1) [reply src/aat/resources/features/F-056/S-582.td.json]: Done...please hav a look

File: src/aat/resources/features/F-056/S-582.td.json (hunk 2 above)
Comment (nitinsahu1) [reply src/aat/resources/features/F-056/S-582.td.json]: Done...please hav a look

File: src/aat/resources/features/F-056/S-582.td.json (hunk 3 above)
Comment (nitinsahu1) [reply src/aat/resources/features/F-056/S-582.td.json]: Done...please hav a look

**Pull request merged**